src/js/edu-benefits/1990/config/migrations.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash/fp';
+
+export default [
+  // 0 -> 1, we have split the edu bundle into form-specific bundles with a new urlPrefix
+  // and this replaces the saved returnUrl if necessary
+  (savedData) => {
```

Comment:
Can we put this function in a common location somewhere and pass in the form number, so it isn't duplicated for each form?

Reply:
Good point, change made.